boltdesignsystem/bolt
# Migrate Icon component over to LitElement; reduce overall SVG file size by 2.5x

## Jira
http://vjira2:8080/browse/BDS-2127

## Summary
Switches our Icon component over to render via LitElement + significantly reduce SVG file sizes by updating (and simplifying!) the Icon-portion of our build process.

Updates the Jest snapshots of components with inlined SVG assets to reflect the now smaller file sizes.

## Details
Before our Icon JS bundle clocked in at around `390.7KB` (uncompressed) -- now it's down to around `146KB`! 🎉 

## How to test
- Reference icon docs to confirm everything continuing to work exactly as expected
- Double-check that Icons render correctly in Accordion (since that's one of the few places where Icons render inside the Shadow DOM of the parent component)
